pkg/limayaml/default.yaml

```diff
@@ -45,6 +45,8 @@ mounts:
     # SSHFS has an optional flag called 'follow_symlinks'. This allows mounts
     # to be properly resolved in the guest os and allow for access to the
     # contents in the symlnked folder. This defaults to false if not supplied.
+    # As a result, symlinked folders on the Host system will look and feel like
```

Comment:
It is both files and directories, not just directories. Please also mention that symlinks to directories cannot be deleted inside the guest.